content/_index.pt.md

```diff
@@ -0,0 +1,155 @@
+{{% /flexrow %}}
+{{% /div %}}
+{{< div class="codesample" >}}
+
+{{% zigdoctest "assets/zig-code/index.zig" %}}
+
+{{< /div >}}
+{{< /flexrow >}}
+
+
+{{% div class="alt-background" %}}
+{{% div class="container"  style="display:flex;flex-direction:column;justify-content:center;text-align:center; padding: 20px 0;" title="Community" %}}
+
+{{< flexrow class="container" style="justify-content: center;" >}}
+{{% div style="width:25%" %}}
+<img src="https://raw.githubusercontent.com/ziglang/logo/master/ziggy.svg" style="max-height: 200px">
+{{% /div %}}
+
+{{% div class="community-message" %}}
+# A comunidade Zig é descentralizada 
+Qualquer pessoa é livre para começar e manter seu próprio espaço para que a comunidade se reúna.  
+Não existe o conceito de "oficial" ou "não-oficial", entretanto, cada local de reunião tem seus próprios moderadores e regras.
+
+<div style="">
+<h1>
+	<a href="https://github.com/ziglang/zig/wiki/Community" class="button" style="display: inline;">Veja todas as comunidades</a>
+</h1>
+</div>
+{{% /div %}}
+{{< /flexrow >}}
+<div style="height: 50px;"></div>
+
+{{< flexrow class="container" style="justify-content: center;" >}}
+{{% div class="main-development-message" %}}
+# Principal desenvolvimento
+O repositório Zig pode ser encontrado em [https://github.com/ziglang/zig](https://github.com/ziglang/zig), onde também discutimos sobre os problemas e propostas.  
+Espera-se que os contribuidores sigam o Zig [Código de Conduta](https://github.com/ziglang/zig/blob/master/CODE_OF_CONDUCT.md).
+{{% /div %}}
+{{% div style="width:40%" %}}
+<img src="https://raw.githubusercontent.com/ziglang/logo/master/zero.svg" style="max-height: 200px">
+{{% /div %}}
+{{< /flexrow >}}
+{{% /div %}}
+{{% /div %}}
+
+
+{{% div class="container" style="display:flex;flex-direction:column;justify-content:center;text-align:center; padding: 20px 0;" title="Zig Software Foundation" %}}
+## A ZSF é uma corporação sem fins lucrativos 501(c)(3).
+
+A Fundação Zig Software é uma corporação sem fins lucrativos fundada em 2020 por Andrew Kelley, o criador do Zig, com o objetivo de apoiar o desenvolvimento da linguagem. Atualmente, a ZSF é capaz de oferecer trabalho remunerado a taxas competitivas para um pequeno número de colaboradores principais. Esperamos ser capazes de estender esta oferta a mais colaboradores centrais no futuro.
+
+A Zig Software Foundation é sustentada por doações.
+
+<h1>
+	<a href="zsf/" class="button" style="display:inline;">Mais informações</a>
+</h1>
+{{% /div %}}
+
+
+{{< div class="alt-background" style="padding: 20px 0;">}}
+{{% div class="container" title="Sponsors" %}}
+# Patrocinadores corporativos 
+As seguintes empresas estão fornecendo suporte financeiro direto para a fundação Zig Software.
+
+{{% sponsor-logos "monetary" %}}
+ <a href="https://pex.com" rel="noopener nofollow" target="_blank"><picture>
+   <picture>
+     <source srcset="pex-white.svg" media="(prefers-color-scheme: dark)">
+     <img src="pex-dark.svg">
```

Comment:
Hello!
I think there is going to be a problem with loading the SVG (it's a static asset served at `/`).
Shouldn't it be

```html
     <source srcset="../pex-white.svg" media="(prefers-color-scheme: dark)">
     <img src="../pex-dark.svg"
```

or maybe an absolute path (a little more future-proof?)

```html
     <source srcset="/pex-white.svg" media="(prefers-color-scheme: dark)">
     <img src="/pex-dark.svg"
```
?

Reply:
Yes, that is correct, I recommend using the absolute path (e.g. "/pex-dark.svg"). This was left in a bit of an inconsistent state because of the testing done on GitHub pages, which forced us into a subpath. 

Now that we're on the final host we don't have this problem anymore and absolute links to static content are the way to go.

Reply:
Done! [680df3e](https://github.com/ziglang/www.ziglang.org/pull/96/commits/680df3e260651a3134a206821b86777ac565889e)